test/support/policy_helpers.h

```diff
@@ -0,0 +1,47 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+#ifndef _POLICY_HELPERS_H
+#define _POLICY_HELPERS_H
```

Comment:
Do we really need one more support header? Why cannot we add the new stuff to `support/utils.h`, for example?

Reply:
Fixed: file removed.